src/dynsight/_internal/track/track.py

```diff
@@ -0,0 +1,125 @@
+import logging
+from pathlib import Path
+
+import pandas as pd
+import trackpy as tp
+
+from dynsight.trajectory import Trj
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s | %(levelname)s | %(message)s",
+)
+logger = logging.getLogger(__name__)
+
+
+def track_xyz(
```

Comment:
There are some interesting parameters here, which probably need a bit more of an explanation of their ranges and their meaning. I see your doc string, but I think it needs more.

For all these:
  
- [ ]  search_range: float = 10,
- [ ]     memory: int = 1,
- [ ]     adaptive_step: float = 0.5,
- [ ]     adaptive_stop: float = 0.95,

Reply:
How big can each be? What are appropriate ranges?

Reply:
So how does the user select what values to use?